### DISCUSSION TITLE
How to transform automatically basic interactions into MIM interactions?

### DISCUSSION THREAD

**Discussion by JuliaBatUM** [wikipathways/wikipathways-help#56] [Show and tell]
Hi!
I am currently working with PathVisio to curate metabolic reactions, but instead of using MIM interactions, I used basic interactions. Is there a way to replace all the basic interactions with MIM interactions, knowing that I already added all the references to the reactions (in the basic interactions)?
Thank you very much for your help,
Julia

PS: I didn´t know where to ask for help, i hope this is the right place

**Comment by khanspers** [comment]
Hi @JuliaBatUM! Thank you for your question, I transferred it here from wikipathways-faq since this is our main repo for discussions and questions about WikiPathways curation.

There isn't an automated way to convert interactions, but there's an easy solution. It relies on changing the properties of a number of existing interactions at the same time. The interaction, and any associated references, is still there just with different properties.

1. Select the interactions you want to update. You can select more than one interaction by holding down the **Command key** while clicking to select interactions. Alternatively, you can select all interactions first by right-clicking anywhere on the canvas and clicking **Select > Select all Interactions**, and then de-select the ones you don't want using the Command click strategy. 
2. With the interactions selected, go to the **Properties** tab on the right and in the **End Line Type** field, update to the desired mim-interaction type. (The value might be listed as **Different Values** before you update; thats ok and due to having different interaction types selected). 

That should work!

Let me know if you have any questions.

**Comment by JuliaBatUM** [comment]
Hi,

Thank you very much for your help. I´ll try that 😊

Kind regards,

Julia


________________________________
Von: Kristina Hanspers ***@***.***>
Gesendet: Donnerstag, 10. August 2023 21:24:43
An: wikipathways/wikipathways-help
Cc: Bous, Julia (MACSBIO); Mention
Betreff: Re: [wikipathways/wikipathways-help] How to transform automatically basic interactions into MIM interactions? (Discussion #56)


Hi @JuliaBatUM<https://github.com/JuliaBatUM>! Thank you for your question, I transferred it here from wikipathways-faq since this is our main repo for discussions and questions about WikiPathways curation.

There isn't an automated way to convert interactions, but there's an easy solution. It relies on changing the properties of a number of existing interactions at the same time. The interaction, and any associated references, is still there just with different properties.

  1.  Select the interactions you want to update. You can select more than one interaction by holding down the Command key while clicking to select interactions. Alternatively, you can select all interactions first by right-clicking anywhere on the canvas and clicking Select > Select all Interactions, and then de-select the ones you don't want using the Command click strategy.
  2.  With the interactions selected, go to the Properties tab on the right and in the End Line Type field, update to the desired mim-interaction type. (The value might be listed as Different Values before you update; thats ok and due to having different interaction types selected).

That should work!

Let me know if you have any questions.
